styles/styles.css

```diff
@@ -0,0 +1,337 @@
+@import url('https://fonts.googleapis.com/css?family=Archivo+Black|Merriweather:400,400i');
+
+.flex-container {
+    display: flex;
+    /*Need to decide on columns, rows, or other approach*/
+    /*May also need to figure out if you can do multiple grids in one page and if so whether that is best approach here*/
+}
+
+a:link {
+  text-decoration: none;
+}
+  
+a:visited {
+  text-decoration: none;
+}
+  
+#banner {
+  background-color: #093b60;
+  height: 18em;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+}
+
+a.subtitle:visited {
+  color: #093b60;
+}
+
+#letterhead {
+  display: flex;
+  flex-direction: column;
+  
+  margin-top: 3%;
+  color: #81B642;
+  font-family: Merriweather, Times, serif;
+  font-style: italic;
+  text-align: center;
+  font-size: 150%;
+}
+
+.filler {
+  width: 0%;
+  
+ 
+}
+
+#toplogo {
+  max-width: 100%;
+  height: auto;
+}
+
+#cart {
+  display: flex;
+  flex-direction: column;
+  justify-content: right;
+  margin-top: 2rem;
+   
+}
+
+.top-cart {
+  color: white;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+}
+
+
+#carticon {
+/*Size to make small; I've just shrunk it a bit for now*/
+width: 1.5em;
+
+}
+
+.checkout-button {
```

Comment:
Watch your indenting. the properties below should align with the / in the comment.

.checkout-button {
  /* Green background...*/
  background-color: red;
}